data/custom_costs.csv

```diff
@@ -0,0 +1,13 @@
+planning_horizon,technology,parameter,value,unit,source,further description
+all,solar,marginal_cost,0.01,EUR/Mwh,Default,
+all,onwind,marginal_cost,0.015,EUR/Mwh,Default,
+all,offwind,marginal_cost,0.015,EUR/Mwh,Default,
+all,hydro,marginal_cost,0,EUR/Mwh,Default,
+all,H2,marginal_cost,0,EUR/Mwh,Default,
+all,electrolysis,marginal_cost,0,EUR/Mwh,Default,
+all,fuel cell,marginal_cost,0,EUR/Mwh,Default,
+all,battery,marginal_cost,0,EUR/Mwh,Default,
+all,battery inverter,marginal_cost,0,EUR/Mwh,Default,
+all,home battery storage,marginal_cost,0,EUR/Mwh,Default,
+all,water tank charger,marginal_cost,0.03,EUR/Mwh,Default,
+all,central water pit charger,marginal_cost,0.025,EUR/Mwh,Default,
```

Comment:
* "Default" as "further description" is a bit unclear.
* The unit should be `MWh`, not `Mwh`